archive/Thematic/2021/Februrary/progressed/bridgecrew2/template.html

```diff
@@ -719,78 +719,61 @@ <h1 class="null" style="text-align: center;display: block;margin: 0;padding: 0;c
 
 															<td valign="top" class="mcnTextContent" style="padding: 0px 18px 9px;line-height: 150%;mso-line-height-rule: exactly;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;word-break: break-word;color: #111111;font-family: 'Source Sans Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;font-size: 18px;text-align: left;">
 
-																xxx
 
-															</td>
```

Comment:
I cant accept this part. all stuff is necessary here. 
Our task here only to put content from empty.html into <td></td> rows that empty

Reply:
which part? I just copied the content from `empty.html` though?

Reply:
but on github previewer it looks like you deleted some rows

Reply:
this is what i see

![image](https://user-images.githubusercontent.com/1469198/108055049-1f548d80-7058-11eb-9fe0-fedf2a579ada.png)

Reply:
Those are the dummy rows I think from the template? should I not delete them? I thought those are for reference rows only? the values of those rows are just `XXXX`

Reply:
Yep. XXX is a placeholder. you need to replace `xxx` with piece of content from empty.html

if I miss something else that you deleted - please tell me - I'm very scared right now

Reply:
other that those placeholders. I deleted nothing else.

Reply:
this template is so fragile. But soon or later we'll get into a position where it wouldn't be a problem at all

Reply:
I'll keep that in mind.

Reply:
no prob, you can actually see why i'm using pull requests and code reviews here :)

plus, I was ready that we'll need some time, in order to set up our collaboration workflow.

Reply:
buzz when with updated version. I think the easiest way will be to get a clean template and import things from scratch

Reply:
so should I put back all those placeholders? or is there anything else that I'm missing to update?

Reply:
i dont know - show me the code first :)

Reply:
I don't think I need to change anything from this though. 🤷‍♂️

Reply:
well, its not accepted then

Reply:
I'm confused here. I thought that the update is good since the deleted part is just placeholders?

Reply:
we are going in cycles.
easy way

0. just delete this file - its doomed and eating our time
1. copy an empty template from other folders
2. move title
3. move preview
4. split content from empty.html at 3 or 4 parts and paste at places with xxx
5. remove xxx
6. apply ad stuff - but I need to show you where.